src/main/java/com/flightstats/hub/dao/aws/writeQueue/WriteQueueConfigProvider.java

```diff
@@ -0,0 +1,13 @@
+package com.flightstats.hub.dao.aws.writeQueue;
+
+import com.flightstats.hub.app.HubProperties;
+import com.google.inject.Provider;
+
+public class WriteQueueConfigProvider implements Provider<WriteQueueConfig> {
```

Comment:
I personally like this pattern (I may have done something similar somewhere...) and even if there only ends up being two attributes here, I think it's an acceptable size. 👍